### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_ExtendyourdataStudio_Keyboard:The install button tool tip is not hoverable. 

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#3155]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:-A11y_AzureDataStudioStayHealthy;-A11ySev2;-A11yTCS;-A11yWCAG2.1;-NonBenchmark-DesktopApp;-MAS1.4.13;-SQL Azure Data Studio;-FTP;-Keyboard;-STP;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.4.13

**Repro Steps:**
1 . Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 . Navigate Extend your data Studio on Welcome screen.
4 . Navigate to install button 

**Actual:**
On hovering over the button install the tooltip appears but on moving to the tooltip it disappears.

**Expected:**
When hovered over the tooltip the tooltip should be hoverable and not dismiss.

**User Impact**:
Keyboard user on trying to move to the tooltip won't be able to view it since it disappears.

**Attachment link for Reference:**
